core-bundle/src/Controller/BackendPreviewSwitchController.php

```diff
@@ -169,7 +169,7 @@ private function getMembersDataList(BackendUser $user, Request $request): array
             WHERE
                 username LIKE ? $andWhereGroups
                 AND login=1
-                AND disable!=1
+                AND disable=0
```

Comment:
```suggestion
                AND !disable
```
How about using boolean expressions?

Reply:
Does this work in all MySQL versions?

Reply:
Yes (tested with 5.5 and up)
https://www.db-fiddle.com/f/66jye1UoJzVkL52qdrySne/0

Reply:
Yes. To me this would be the nicest syntax, not sure if it is the fastest though.

Reply:
Veryfied: https://www.db-fiddle.com/f/sskh7B7ZRiRCSeM92NZUep/1
We should use `=1` and `=0` for best performance.

So my suggestion needs to be reverted (sorry 🙈).

Reply:
No biggie. I have reverted the changes.